0015-ilp-addresses/0015-ilp-addresses.md

```diff
@@ -54,105 +61,102 @@ The allocation scheme is the first part of an address, which indicates how the a
 
-The general allocation scheme does not allow you to make any assumptions about the meaning of the segments. Segments in the same place could have different meanings to different ledgers or connectors. To make routing work well, we recommend placing the segments in the following order:
+The global allocation scheme does not allow you to make any assumptions about the meaning of the segments. Segments in the same place could have different meanings to different ledgers or connectors. However, to make routing work well, we recommend placing segments in the following order:
 
 - [Neighborhoods](#neighborhoods)
-- [Payment rail](#payment-rail)
-- [Ledger prefix](#ledger-prefix)
-- [Subledger and account group](#subledger-and-account-group)
-- [Account identifier](#account-identifier)
+- [Ledger prefixes](#ledger-prefixes)
+- [Account identifiers](#account-identifiers)
 - [Interactions](#interactions)
 
 Not all addresses contain all this information, and some addresses may use multiple segments to represent some of this information.
 
 ### Neighborhoods
 
-Neighborhoods have no specific meaning, but serve as a quick shortcut for routing to the right area.
+Neighborhoods have no specific meaning, but serve as a quick shortcut for routing to the right area. At this time, there is no official list of neighborhoods, but the following list of examples should illustrate what might constitute a neighborhood:
+
+- `crypto.` for ledgers related to decentralized crypto-currencies such as Bitcoin, Etherium, or XRP.
+- `sepa.` for ledgers in the [Single Euro Payments Area](https://en.wikipedia.org/wiki/Single_Euro_Payments_Area).
+- `dev.` for Interledger Protocol development and early adopters
 
-- For regional currencies, use a continent code from the following table:
-    | Segment | Continent     |
-    |:--------|:--------------|
-    | `af`    | Africa        |
-    | `as`    | Asia          |
-    | `eu`    | Europe        |
-    | `na`    | North America |
-    | `sa`    | South America |
-    | `oc`    | Oceania       |
-    | `an`    | Antarctica    |
-- For decentralized crypto-currency ledgers, use the `crypto` neighborhood.
-- For naturally-ocurring commodities or other globally-distributed currencies, use the `earth` neighborhood.
-- For currencies strongly associated with a particular international corporation, use the continent of that corporation's headquarters.
+Large neighborhoods can be further subdivided into nested sub-neighborhoods; for example, `dev.luxembourg.` for development ledgers hosted in Luxembourg.
```

Comment:
I would also mention that the point of neighborhoods is to group parties such that everyone in the same neighborhood is likely connected. If everyone in `dev` knows about everyone else in `dev`, then it's a useful neighborhood. If not, but everyone in `dev.luxembourg` is connected, then that's a more useful neighborhood.